sips/sip-030/sip-030-wallet-interface.md

```diff
@@ -0,0 +1,528 @@
+# Preamble
+
+SIP Number: `To be assigned`
+
+Title: Integration of a Modern Stacks Wallet Interface Standard
+
+Authors: [janniks](https://github.com/janniks), [kyranjamie](https://github.com/kyranjamie), [aryzing](https://github.com/aryzing), [m-aboelenein](https://github.com/m-aboelenein)
+
+Type: Standard
+
+Status: Draft
+
+Created: 10 October 2023
+
+License: BSD 2-Clause
+
+# OPEN QUESTIONS
+
+<!-- todo: remove section before merge -->
+
+- [ ] Should a global single `window.` object be used, or should provider discovery be handled complely differently?
+- [ ] Rename `stx_transferFt` to include SIP010 somewhere?
+- [ ] Should listeners have params? (e.g. txid for tx mined event)
+
+# Abstract
+
+This SIP proposes the integration of the standard into the Stacks blockchain's "Connect" and "Auth" system.
+The goal is to replace the current Connect interface, primarily used by web applications to connect with browser extensions and mobile apps, with a more straightforward protocol.
+The proposal consists mainly of standardizing JSON compatible interfaces for use with wallet interfaces.
+
+# Motivation
+
+The current Connect system, which has evolved since the Blockstack era, is primarily utilized by web applications for interfacing with wallets.
```

Comment:
Unless this will be annotated or linked, i would change "which has evolved since the Blockstack era" to something more generic like: "which has existed for several years, ..."
or, remove that part of the sentence altogether unless it's more relevant later on (still reading)